src/sagemaker_training/torch_distributed.py

```diff
@@ -94,7 +94,13 @@ def _create_command(self):
         if entrypoint_type is _entry_point_type.PYTHON_PROGRAM:
             num_hosts = len(self._hosts)
             torchrun_cmd = []
-
+            
+            """
```

Comment:
generally this comment style is used for function but not single lines, do you mind update it to "#"-started comment?